source/reference/limitations.txt

```diff
@@ -89,6 +89,8 @@ Sharded Clusters
   to a replica set.
 - ``mongosync`` doesn't support sync to a sharded cluster 
   topology with one or more arbiters.
+- ``mongosync`` doesn't support sync to or from :ref:`global clusters 
+  <global-clusters>`
```

Comment:
[typo]

This needs a full stop at the end.

Reply:
Ack! Thanks for catching that. Fixed.